### PR TITLE
[Governance]: Deliverables lifecycle

### PR DESCRIPTION
**Pull request by tkuhrt**

### CODE REVIEW

File: docs/governance/paper-lifecycle.md
Comment (tlodderstedt) [inline docs/governance/paper-lifecycle.md:40]: I would welcome a broader review process open to the whole foundation.

Comment (tkuhrt) [reply docs/governance/paper-lifecycle.md:40]: I have attempted to make this clearer in the information note following this paragraph.

> Comments and feedback can be provided by anyone in the OWF community. The TAC will take into consideration any comments and feedback provided during the review process when making their decision.

Comment (tlodderstedt) [reply docs/governance/paper-lifecycle.md:40]: thanks!

**Comment by wenjing** [comment]
Tracy, one larger issue that the current process doc does not currently address is the nature of the "paper" which I think is important for OWF. What does a "published paper" that obtained a TAC approval mean? Does it somehow mean the document has become a consensus of the overall OWF community? And how should we present to our audience so that they understand it correctly? Clarity in these questions would answer potential concerns and also help TAC members to make sound and transparent decisions.

**Comment by tkuhrt** [comment]
> Tracy, one larger issue that the current process doc does not currently address is the nature of the "paper" which I think is important for OWF. What does a "published paper" that obtained a TAC approval mean? Does it somehow mean the document has become a consensus of the overall OWF community? And how should we present to our audience so that they understand it correctly? Clarity in these questions would answer potential concerns and also help TAC members to make sound and transparent decisions.

Hi, @wenjing. I made some changes to the document. First, I changed "papers" to "deliverables". I then added some examples of different types of deliverables that we might expect our task forces or special interest groups to create. I am hoping that these changes address the concerns that people might have.

**Review by skounis — APPROVED** [review]

File: docs/governance/deliverable-lifecycle.md
Comment (davidejalexander) [inline docs/governance/deliverable-lifecycle.md:1]: I attended the last TAC where this was discussed and there was some opposition to the idea of publishing documents and reports by a TAC Voting member.  I was really confused by the conversation. We are an open community producing open source code and collaborating on key issues that will help the OWF deliver its mission. I am fully supportive of this process and publishing a wide range of deliverables

If the objection was about deliverables being too prescriptive or mandated I think this was a misunderstanding by some people.

This is sharing knowledge and experience and the outputs from collaborative work an in some cases reporting on progress and findings packaged up for onward consumption and review by anyone. The Safe Wallet Guide is one such example as is the Wallet UX document in development

If I have missed something would those who expressed concern please explain the concern and also why this was something that one person suggested needed and OWF Board approval.

Comment (wenjing) [reply docs/governance/deliverable-lifecycle.md:1]: I do agree it is about the potential for misunderstandings, so the concern is how we may reduce such misunderstandings esp when documents are distributed more widely. All WGs/SIGs/projects in OWF can author and publish documents in any way they see beneficial to their tasks/goals, I don't recall that needs TAC approval. So the fact that some documents request TAC approval seems to suggest there are something special about them, or maybe we want to confer an "official" status representing some kind of community wide consensus, or to use OWF logos to indicate so to all readers, etc. etc. Whatever "it" may be, I thought clarifying that meaning will be helpful to reduce the type of "misunderstanding" you mentioned. OWF has often stated that we will not be a standard organization, and I do *not* believe the proposal here is contrary to that position, however the potential for others to misunderstand is there. So it's a good opportunity for us to be clear about it in this lifecylce document. And, preferably, may also use some simple text to indicate the document's status in the header sections, as a practical suggestion.

Secondly, I am also interested in knowing what TAC members should do in approving or rejecting a deliverable if one comes for TAC review. I understand there will be procedural criteria, e.g. is it properly formatted with proper disclaimers, copyright notices etc. If it's more than that, then I'm wondering what types of evaluations a TAC member is supposed to do to form an opinion. Do they have to agree with the essence of the document? Is that something we like our TAC to do? There will be pros and cons in getting TAC involved. After all, TAC members may not be domain experts in the content being reviewed or have bandwidth for reviewing many such documents if the volume becomes large. Again, this second concern is  an extension of the first one. If we know the exact intention, we would be better at setting guidelines that TAC members can follow. 

Hope that clarifies. These concerns are not for restricting publication in anyway, in fact, it could make publication a lot easier if we clarify the currently vague status and associated criteria.

Comment (wenjing) [reply docs/governance/deliverable-lifecycle.md:1]: By the way, we heard the need to publish the "Safe Wallet Guide" as soon as possible in today's TAC meeting. I think we could take a practical step to approve that first, and maybe spend a little more time to sort out what this lifecycle document needs to be. If we do make changes here, we could always ask a minor revision to the Safe Wallet Guide later. There is no reason to hold up their good work in my view.

Comment (tkuhrt) [reply docs/governance/deliverable-lifecycle.md:1]: Let me start by saying that I agree that the original task force and SIG governance did not mention anything about the work being done by these groups needing to be approved by the TAC only that the task forces and SIGs provide updates on a quarterly basis to the TAC. I was under the impression that people in the community wanted there to be a formal approval process by the TAC for any material produced by task forces and SIGs. If this is not the case, then we can close this PR. This means that there will be no process for approval of any deliverables that are created by the task forces and SIGs that have been approved by the TAC.

In my opinion, what we are looking to “approve” is whether the task force and SIG delivered on their stated goals. There are others who I have heard are more concerned with the content of those deliverables being approved. Thus this PR.

Comment (skounis) [reply docs/governance/deliverable-lifecycle.md:1]: If we intend to establish an approval process, we must define precise criteria upfront. 

Will these criteria focus on editorial aspects or substance evaluation? Regarding the latter, as @wenjing has pointed out, it's unclear whether TAC members possess the capacity to perform such evaluations. 

Additionally, I support implementing a process that checks the editorial aspect to ensure uniformity across all our documents.

**Review by skounis — APPROVED** [review]

File: docs/governance/deliverable-lifecycle.md
Comment (tkuhrt) [inline docs/governance/deliverable-lifecycle.md:99]: It was pointed out to me that the CC-by-4.0 license already has a disclaimer section. Maybe we should remove this. Text from [CC-by-4.0](https://creativecommons.org/licenses/by/4.0/legalcode.en):

> Section 5 – Disclaimer of Warranties and Limitation of Liability.
>
> a. Unless otherwise separately undertaken by the Licensor, to the extent possible, the Licensor offers the Licensed Material as-is and as-available, and makes no representations or warranties of any kind concerning the Licensed Material, whether express, implied, statutory, or other. This includes, without limitation, warranties of title, merchantability, fitness for a particular purpose, non-infringement, absence of latent or other defects, accuracy, or the presence or absence of errors, whether or not known or discoverable. Where disclaimers of warranties are not allowed in full or in part, this disclaimer may not apply to You.
>
> b. To the extent possible, in no event will the Licensor be liable to You on any legal theory (including, without limitation, negligence) or otherwise for any direct, special, indirect, incidental, consequential, punitive, exemplary, or other losses, costs, expenses, or damages arising out of this Public License or use of the Licensed Material, even if the Licensor has been advised of the possibility of such losses, costs, expenses, or damages. Where a limitation of liability is not allowed in full or in part, this limitation may not apply to You.
>
> c. The disclaimer of warranties and limitation of liability provided above shall be interpreted in a manner that, to the extent possible, most closely approximates an absolute disclaimer and waiver of all liability.

Comment (tkuhrt) [reply docs/governance/deliverable-lifecycle.md]: Removed this section and added a non-representative disclaimer section.

**Review by skounis — APPROVED** [review]

**Review by skounis — APPROVED** [review]

**Comment by tkuhrt** [comment]
Possible language for the disclaimer:
> ...do not necessarily reflect the official policies, positions, views or opinions of...

**Comment by tkuhrt** [comment]
Possible alternate language to replace the disclaimer:

> These materials are a deliverable of the OWF community. If you have concerns or suggestions regarding the content, please file a pull request.

**Comment by tkuhrt** [comment]
**Approved during the September 4, 2024 meeting**

**Pull request merged**
